src/test/resources/deployments/send_deployment_result.json

```diff
@@ -0,0 +1,3 @@
+{
+  // It actually doesn't matter what the response is because we never read it. We only check status code.
```

Comment:
could we put in the real response regardless? That would enable us to look at the response and perhaps do some validation (should we want to in the future) without first having to find the actual response. Just a thought